be/src/exec/tablet_sink_sender.h

```diff
@@ -78,6 +78,14 @@ class TabletSinkSender {
         }
     }
 
+    void _update_partition_rows(int64_t partition_id) {
+        if (auto iterator = _partition_to_num_rows.find(partition_id); iterator != _partition_to_num_rows.end()) {
```

Comment:
nit: we should test the performance when num_rows=1bilion and partition number is 2000